operations/system_management_health/victoriametrics.md

```diff
@@ -0,0 +1,96 @@
+# VictoriaMetrics
+
+VictoriaMetrics is a fast, cost-effective and scalable time series database. It can be used as a long-term remote storage for Prometheus.
```

Comment:
```suggestion
VictoriaMetrics is a fast, cost-effective, and scalable time series database. It can be used as a long-term remote storage for Prometheus.
```